src/components/ebay-date-textbox/component.ts

```diff
@@ -66,7 +66,7 @@ class DateTextbox extends Marko.Component<Input, State> {
     }
 
     onInput(input: Input) {
-        if (input.value) {
+        if (input.value !== undefined) {
```

Comment:
Do we need to account for empty strings in this conditional?

Reply:
No, we shouldn't need to. An empty string would also work to clear the date textbox, but devs shouldn't be passing any strings that don't work as ISO strings.